### ISSUE TITLE
Crash on iOS simulator 16.0 (1036)

### ISSUE THREAD

**Issue by knopp** [rive-app/rive-flutter#412]
Rive animations with audio crash on iOS simulator 16.0 (1036), regardless of iOS version running in the simulator.

```
Assertion failed: (pDevice->playback.internalChannels <= 254), function ma_on_output__coreaudio, file miniaudio.h, line 33508.
```

Stacktrace
```
#0	0x00000001029c1008 in __pthread_kill ()
#1	0x000000010320b408 in pthread_kill ()
#2	0x000000018012873c in abort ()
#3	0x0000000180127bb0 in __assert_rtn ()
#4	0x0000000102cb150c in ma_on_output__coreaudio.cold.3 at /Users/Matej/Projects/superlist1/packages/superlist/ios/.symlinks/plugins/rive_common/ios/miniaudio/miniaudio.h:33508
#5	0x0000000102b8ab30 in ma_on_output__coreaudio at /Users/Matej/Projects/superlist1/packages/superlist/ios/.symlinks/plugins/rive_common/ios/miniaudio/miniaudio.h:33508
#6	0x0000000197a45674 in ausdk::AUInputElement::PullInput ()
#7	0x00000001979941ac in AUInputFormatConverter2::InputProc ()
#8	0x000000018b8797d4 in acv1::AudioConverterChain::CallInputProc ()
#9	0x000000018b87af68 in acv1::AudioConverterChain::FillBufferFromInputProc ()
#10	0x000000018b9ae680 in acv1::BufferedAudioConverter::GetInputBytes ()
#11	0x000000018b92363c in acv1::Resampler2Wrapper::RenderOutput ()
#12	0x000000018b9f6c40 in acv1::SampleRateConverter::RenderOutput ()
#13	0x000000018b9ae428 in acv1::BufferedAudioConverter::FillBuffer ()
#14	0x000000018b879578 in acv1::AudioConverterChain::RenderOutput ()
#15	0x000000018b9ae428 in acv1::BufferedAudioConverter::FillBuffer ()
#16	0x000000018b9e5888 in acv1::_AudioConverterFillComplexBuffer ()
#17	0x0000000197992b90 in AUConverterBase::RenderBus ()
#18	0x0000000197a02240 in AURemoteIO::RenderBus ()
#19	0x0000000197a4a550 in ausdk::AUBase::DoRender ()
#20	0x0000000197a0c3ac in AURemoteIO::PerformIO ()
#21	0x0000000197a41fd0 in _XPerformIO ()
#22	0x000000018a09bafc in mshMIGPerform ()
#23	0x000000018a09bef0 in MSHMIGDispatchMessage ()
#24	0x0000000197a04e98 in caulk::thread_proxy<std::__1::tuple<caulk::thread::attributes, AURemoteIO::IOThread::IOThread(AURemoteIO&, caulk::thread::attributes const&, caulk::mach::os_workgroup_managed const&)::'lambda'(), std::__1::tuple<>>> ()
#25	0x000000010320b6f8 in _pthread_start ()
```

**Comment by knopp** [comment]
Edit: Also happens in 1037 (just released): SimulatorKit 942, CoreSimulator 987

**Comment by HayesGordon** [comment]
Thanks for reporting, this should now be resolved in v0.13.15
